gladier_xpcs/tools/xpcs_boost_solver.py

```diff
@@ -0,0 +1,82 @@
+from gladier import GladierBaseTool, generate_flow_definition
+
+def xpcs_boost(**data):
+    import os
+    import json
+
+    ##minimal data inputs payload
+    proc_dir = data.get('proc_dir') # location of transfered files / results
+    raw_file = data.get('raw_file') # raw data
+    qmap_file = data.get('qmap_file') # name of the qmap file
+    atype = data.get('atype','Both') #["Multitau", "Twotime", "Both"]
+    config_file = data.get('config_file', None) # name of the config file
+    gpu_flag = data.get('gpu_flag',0)
+    verbose = data.get('verbose', False)
+
+    if not os.path.exists(proc_dir):
+        raise NameError(f'{proc_dir} \n Proc dir does not exist!')
+
+    os.chdir(proc_dir)
+
+    # if config_file:
+    #     with open(config_file) as f:
+    #         config = json.load(f)
+
+    if atype == 'Multitau' or atype == 'Both':
+        from boost_corr.xpcs_aps_8idi.gpu_corr_multitau import solve_multitau
+        solve_multitau(qmap=qmap_file,
+                    raw=raw_file,
+                    output=proc_dir,
+                    batch_size=8,
+                    gpu_id=gpu_flag,
+                    verbose=verbose,
+                    masked_ratio_threshold=0.75,
+                    use_loader=True,
+                    begin_frame=1,
+                    end_frame=-1,
+                    avg_frame=1,
+                    stride_frame=1,
+                    overwrite=False)
+
+    if atype == 'Twotime' or atype == 'Both':
+        from boost_corr.xpcs_aps_8idi.gpu_corr_twotime import solve_twotime
+        solve_twotime(qmap=qmap_file,
+                    raw=raw_file,
+                    output=proc_dir,
+                    batch_size=8,
+                    gpu_id=gpu_flag,
+                    verbose=verbose,
+                    begin_frame=1,
+                    end_frame=-1,
+                    avg_frame=1,
+                    stride_frame=1,
+                    dq_selection=None,
+                    smooth='sqmap')
+
+    return
```

Comment:
Think we could simplify this further and just pass `**data`? Or, maybe we could change the input to include all possible corr options under data and pass a whole object insead. Something like: `**data['corr_boost_kwargs']`
That would allow passing new arbitrary args without us needing to change up the tooling.

Reply:
I think this is a good idea as long as we have a lot of those as default. We can keep the old clients and inputs as much the same as possible. I will investigate that.

Reply:
Additionally, is there any output we can capture? Not sure if these tools return anything, or output to sys.stdout, but anything we could propagate back up to flows would be good.

Reply:
Should we open another PR for tool improvement once this one is tested and upstream?